docs/docs/how-to-deploy.mdx

```diff
@@ -511,10 +479,6 @@ your chosen container registry.
 How you reference the custom action image will depend on your deployment. Pick the relevant documentation for
```

Comment:
The action server docs need to be clear about how you use the action server with the open source helm chart imo